engines/aws/app/models/aws/metadata.rb

```diff
@@ -14,7 +14,7 @@ def self.load
 
     def execute(*args)
       stdout, stderr = Cheetah.run(
-        ['ec2metadata', '--api', 'latest', *args],
+        ['ec2metadata', '--api', '2021-07-15', *args],
```

Comment:
I do not see how this helps
```
~> ec2metadata --api 2021-07-15 --info
{
  "Code" : "Success",
  "LastUpdated" : "2022-10-05T18:30:34Z",
  "AccountId" : "810320120389"
}
```
```
~> ec2metadata --api latest --info
{
  "Code" : "Success",
  "LastUpdated" : "2022-10-05T18:31:12Z",
  "AccountId" : "810320120389"
}
```
Until there is a fix for ec2metadata, i.e. bsc#1204066 I think the only reliable solution is to make the http request in ` def instance_profile_arn` and access the metadata server directly.

Reply:
isn't the aim to have the same version even though `latest` change ? so if `latest` changed to 
whatever version (!= `2021-07-15`), `suse-rancher-setup` would still use `2021-07-15`, right ?